<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editUser</class>
 <widget class="QMainWindow" name="editUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Emad-Diana eXtra</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>المستخدمين</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>210</y>
      <width>821</width>
      <height>641</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
  background-color:#fffff;
  object-fit: contain;
  font-size: 20px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 0.04;
  letter-spacing: normal;
  text-align: center;
 selection-background-color: #0f552f;
  color: black;}</string>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>الكود</string>
     </property>
     <property name="font">
      <font>
       <family>Emad-Diana eXtra</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>المستخدم</string>
     </property>
     <property name="font">
      <font>
       <family>Emad-Diana eXtra</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>الاسم</string>
     </property>
     <property name="font">
      <font>
       <family>Emad-Diana eXtra</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>الملعلومات</string>
     </property>
     <property name="font">
      <font>
       <family>Emad-Diana eXtra</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1010</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/backGround/1920X1030.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>90</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/green rectangle/rectangle-3@3x.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>88</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Emad-Diana eXtra</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 width: 156px;
  height: 26px;
  object-fit: contain;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 0.04;
  letter-spacing: normal;
  text-align: right;
  color: #ffffff;}</string>
    </property>
    <property name="text">
     <string>الأجل</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Emad-Diana eXtra</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
  object-fit: contain;

  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 0.04;
  letter-spacing: normal;
  text-align: center;
  color: #ffffff;
    background-color: #0f552f;
	font
    border-style: solid;
    }

QPushButton:hover {

    }

QPushButton:pressed {
 
    }</string>
    </property>
    <property name="text">
     <string>مسح</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Emad-Diana eXtra</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
  object-fit: contain;

  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 0.04;
  letter-spacing: normal;
  text-align: center;
  color: #ffffff;
    background-color: #0f552f;
	font
    border-style: solid;
    }

QPushButton:hover {

    }

QPushButton:pressed {
 
    }</string>
    </property>
    <property name="text">
     <string>تعديل</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>940</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/copyRights/copyright-2020-cashier-all-rights-reserved@3x.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>920</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/bottom logo/layer-7.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_delete</zorder>
   <zorder>pushButton_edit</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resourceFiles_rc.qrc"/>
 </resources>
 <connections/>
</ui>
